src/extensions/ts-setup.js

```diff
@@ -37,12 +36,19 @@ module.exports = (toolbox) => {
     function syncOperations() {
       if (framework !== 'nest') {
         copy(`${assetsPath}/tsconfig.json`, `${appDir}/tsconfig.json`);
-        pkgJsonScripts.push({
-          ['build']: 'tsc --build && tsc-alias',
-          ['prepare']: 'npm run build',
+
+        Object.assign(pkgJson.scripts, {
+          build: 'tsc --build && tsc-alias',
+          prepare: 'npm run build',
```

Comment:
This will overwrite the `husky` script with the same property name or vice versa, whichever comes second will be the one to stay

Reply:
I'll remove this line.

Reply:
Do we want to have the build step in a `prepare` script or we'll require the developer to call it out specifically before running the app? We can decide on this when we setup the `start` script.